### PR TITLE
Feature/build clang rust

### PR DESCRIPTION
**Pull request by lunn**
Fix type RUSG->RUST
Redirect skip message to description
Fix test if DESC_FD is missing

### CODE REVIEW

**Comment by kuba-moo** [comment]
i may have force pushed at some stage :S Could you rebase?

**Comment by lunn** [comment]
Rebased, and github seems happy now.

File: tests/patch/build_clang_rust/build_clang_rust.sh [hunk 1]
```diff
@@ -60,7 +60,7 @@ prep_config() {
     fi
 }
 
-if [ -n "$DESC_FD"]; then
+if [ -z ${DESC_FD+x} ]; then
```
Comment (kuba-moo) [inline tests/patch/build_clang_rust/build_clang_rust.sh]: I'm not used to this pattern. Is it correct?

```
 ~$ test -z ${DESC_FD+x}
 ~$ echo $?
1
 ~$ DESC_FD=3
 ~$ test -z ${DESC_FD+x}
 ~$ echo $?
1
```

I'm used to `[ ${VAR}x != x ]`

**Comment by lunn** [comment]
On Wed, Nov 01, 2023 at 08:30:02PM -0700, Jakub K wrote:
> @kuba-moo commented on this pull request.
> 
> ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
> 
> In tests/patch/build_clang_rust/build_clang_rust.sh:
> 
> > @@ -60,7 +60,7 @@ prep_config() {
>      fi
>  }
> 
> -if [ -n "$DESC_FD"]; then
> +if [ -z ${DESC_FD+x} ]; then
> 
> I'm not used to this pattern. Is it correct?
> 
>  ~$ test -z ${DESC_FD+x}
>  ~$ echo $?
> 1
>  ~$ DESC_FD=3
>  ~$ test -z ${DESC_FD+x}
>  ~$ echo $?
> 1
> 
> I'm used to [ ${VAR}x != x ]

I'm not really a bash programmer. It worked for me.

But i changed to your pattern.

    Andrew

**Pull request merged**